nat-lab/tests/test_dns_through_exit.py

```diff
@@ -6,16 +6,53 @@
 from contextlib import AsyncExitStack
 from mesh_api import API
 from telio import AdapterType, State
+from typing import List, Tuple
 from utils import testing
 from utils.connection_tracker import ConnectionLimits
 from utils.connection_util import (
     generate_connection_tracker_config,
     ConnectionTag,
     new_connection_with_conn_tracker,
 )
+from utils.router import IPStack
 
 
+# IPv6 tests are failing because we do not have IPV6 internet connection
 @pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "alpha_info",
+    [
+        pytest.param(
+            (IPStack.IPv4, ["1.1.1.1"]),
+            marks=pytest.mark.ipv4,
+        ),
+        # pytest.param(
+        #     IPStack.IPv6,
+        #     marks=pytest.mark.ipv6,
+        # ),
+        pytest.param(
+            (IPStack.IPv4v6, ["1.1.1.1", "2606:4700:4700::1111"]),
+            marks=pytest.mark.ipv4v6,
+        ),
+    ],
+)
+@pytest.mark.parametrize(
+    "exit_info",
+    [
+        pytest.param(
+            (IPStack.IPv4, ["8.8.8.8"]),
+            marks=pytest.mark.ipv4,
+        ),
+        # pytest.param(
+        #     IPStack.IPv6,
+        #     marks=pytest.mark.ipv6,
```

Comment:
Leave a comment why this code is commented :)

Reply:
Done!